pkg/services/gitops/install.go

```diff
@@ -78,13 +80,17 @@ func (g *Gitops) Install(params InstallParams) (map[string][]byte, error) {
 		}
 		for _, m := range wegoAppManifests {
 			if err := g.kube.Apply(ctx, m, params.Namespace); err != nil {
-				return nil, fmt.Errorf("error applying wego-app manifest %s: %w", m, err)
+				return nil, fmt.Errorf("error applying wego-app manifest \n%s: %w", m, err)
 			}
 		}
 
 		systemManifests["wego-app.yaml"] = bytes.Join(wegoAppManifests, []byte("---\n"))
 	}
 
+	if err := g.saveWegoConfig(ctx, params); err != nil {
+		return nil, err
+	}
+
```

Comment:
The wego config should be part of the manifests returned.  That way they show up in the dryrun and will be persisted.